advanced/time-accounting.rst

```diff
@@ -42,3 +42,15 @@ a ticket. Enter how much time you spent on it.
    for, noting that one of the choices may be pre-selected. You can always
    remove the selection for general accounted time that is not supposed to be
    grouped together.
+
+**Where can I see how much time has been accounted to this ticket?**
+
+If a ticket already has accounted time(s), you can see it in the ticket pane
```

Comment:
`ticket pane` sounds wrong to me. It's the ticket sidebar, right? How do we call it in other places?

Reply:
For me as well, but it looks like it is widely used in the docs. "Sidebar" is used just once. I think in this case it would be better to stick to the existing wording, right?

Reply:
Yeah, let's keep it.